src/commands/Configuration/configure/language.ts

```diff
@@ -44,10 +44,15 @@ async function language(interaction: MessageComponentInteraction, client: BulbBo
 			.setStyle("DANGER"),
 	);
 
-	await interaction.update({
-		content: await client.bulbutils.translate("config_language_header", interaction.guild?.id, {}),
-		components: [placeholderRow, selectRow, buttonRow],
-	});
+	interaction.deferred
+		? await interaction.editReply({
+				content: await client.bulbutils.translate("config_language_header", interaction.guild?.id, {}),
+				components: [placeholderRow, selectRow, buttonRow],
+		  })
+		: await interaction.update({
+				content: await client.bulbutils.translate("config_language_header", interaction.guild?.id, {}),
+				components: [placeholderRow, selectRow, buttonRow],
+		  });
```

Comment:
```suggestion
	await interaction[interaction.deferred ? 'editReply' : 'update']({
		content: await client.bulbutils.translate("config_language_header", interaction.guild?.id, {}),
		components: [placeholderRow, selectRow, buttonRow],
	});
```
Same thing can be done wherever else this pattern was used

Reply:
Ty, this looks much cleaner